### ISSUE TITLE
Corpus.load throws exception when using an absolute path?

### ISSUE THREAD

**Issue by mepatterson** [chartbeat-labs/textacy#71]
This could be totally something I'm doing wrong, but I'm stumped. I have a class that creates a topic model using textacy and then saves all the relevant corpus and model files out to disk for use later. When that class is instantiated, it first checks to see if the model files already exist on disk:

```
# /Users/mepatterson/stufftopic/modeler/stuff_topic_model.py
MOD_PATH = os.path.dirname(__file__)

if os.path.exists(os.path.join(MOD_PATH, 'stuff.corpus_spacy_docs.bin')):
    logging.info("Found stuff.corpus. Loading...")
    corpus = textacy.Corpus.load(MOD_PATH, name='stuff.corpus', compression='gzip')
```

When I run this file from within the `modeler` subdir itself, everything works great:
`python analyzer.py`
(analyzer imports the stuff_topic_model class from the same directory)
ALL OK!

But when I run this from the parent subdir, it blows up:
`python modeler/analyzer.py`

```
Traceback (most recent call last):
  File "modeler/analyzer.py", line 19, in <module>
    stm = StuffTopicModel()
  File " /Users/mepatterson/stufftopic/modeler/stuff_topic_model.py", line 35, in __init__
    self.load_corpus()
  File " /Users/mepatterson/stufftopic/modeler/stuff_topic_model.py", line 44, in load_corpus
    corpus = textacy.Corpus.load(MOD_PATH, name='stuff.corpus', compression='gzip')
  File "/Users/mepatterson/anaconda/envs/ironman/lib/python2.7/site-packages/textacy/corpus.py", line 268, in load
    for spacy_doc, metadata in zip(spacy_docs, metadata_stream):
  File "/Users/mepatterson/anaconda/envs/ironman/lib/python2.7/site-packages/textacy/fileio/read.py", line 173, in read_spacy_docs
    yield SpacyDoc(spacy_vocab).from_bytes(bytes_string)
  File "spacy/tokens/doc.pyx", line 615, in spacy.tokens.doc.Doc.from_bytes (spacy/tokens/doc.cpp:12347)
  File "spacy/serialize/packer.pyx", line 129, in spacy.serialize.packer.Packer.unpack_into (spacy/serialize/packer.cpp:6219)
  File "spacy/serialize/packer.pyx", line 184, in spacy.serialize.packer.Packer._char_decode (spacy/serialize/packer.cpp:7598)
UnicodeDecodeError: 'utf8' codec can't decode byte 0x80 in position 5: invalid start byte
```

Any ideas? Is it something I'm doing wrong with relation to file loading from different directories? Or is this a bug in spacy itself?

**Comment by mepatterson** [comment]
In fact, I can demonstrate the bug even simpler. I have the `stuff.corpus` stored in `stufftopic/modeler/`. I am in the `stufftopic` directory. Load iPython.

```
In [1]: import spacy

In [2]: import textacy

In [3]: corpus = textacy.Corpus.load('/Users/mepatterson/stufftopic/modeler', name='stuff.corpus', compression='gzip')
---------------------------------------------------------------------------
UnicodeDecodeError                        Traceback (most recent call last)
<ipython-input-4-8a3e0dd529b3> in <module>()
----> 1 corpus = textacy.Corpus.load('/Users/mepatterson/stufftopic/modeler', name='stuff.corpus', compression='gzip')

/Users/mepatterson/anaconda/envs/ironman/lib/python2.7/site-packages/textacy/corpus.pyc in load(cls, path, name, compression)
    266         metadata_stream = fileio.read_json_lines(meta_fname, mode=meta_mode)
    267         spacy_docs = fileio.read_spacy_docs(corpus.spacy_vocab, docs_fname)
--> 268         for spacy_doc, metadata in zip(spacy_docs, metadata_stream):
    269             corpus.add_doc(
    270                 Doc(spacy_doc, lang=corpus.spacy_lang, metadata=metadata))

/Users/mepatterson/anaconda/envs/ironman/lib/python2.7/site-packages/textacy/fileio/read.pyc in read_spacy_docs(spacy_vocab, filepath)
    171     with open_sesame(filepath, mode='rb') as f:
    172         for bytes_string in SpacyDoc.read_bytes(f):
--> 173             yield SpacyDoc(spacy_vocab).from_bytes(bytes_string)
    174
    175

/Users/mepatterson/anaconda/envs/ironman/lib/python2.7/site-packages/spacy/tokens/doc.pyx in spacy.tokens.doc.Doc.from_bytes (spacy/tokens/doc.cpp:12347)()
    613     def from_bytes(self, data):
    614         '''Deserialize, loading from bytes.'''
--> 615         self.vocab.serializer.unpack_into(data[4:], self)
    616         return self
    617

/Users/mepatterson/anaconda/envs/ironman/lib/python2.7/site-packages/spacy/serialize/packer.pyx in spacy.serialize.packer.Packer.unpack_into (spacy/serialize/packer.cpp:6219)()
    127             self._orth_decode(bits, length, doc)
    128         else:
--> 129             self._char_decode(bits, -length, doc)
    130         array = numpy.zeros(shape=(len(doc), len(self._codecs)), dtype=numpy.int32)
    131         for i, codec in enumerate(self._codecs):

/Users/mepatterson/anaconda/envs/ironman/lib/python2.7/site-packages/spacy/serialize/packer.pyx in spacy.serialize.packer.Packer._char_decode (spacy/serialize/packer.cpp:7598)()
    182         self.char_codec.decode(bits, utf8_str)
    183
--> 184         cdef unicode string = utf8_str.decode('utf8')
    185         cdef int start = 0
    186         cdef bint is_spacy

UnicodeDecodeError: 'utf8' codec can't decode byte 0x80 in position 5: invalid start byte
```

**Comment by mepatterson** [comment]
Closing. As always I figured it out right after posting the "bug."

Turns out to be my fault. I'll leave here for others who see an error like that. It likely means you're not properly loading spacy's data files. I have mine baked into my app directory, so I set a relative path with `spacy.util.set_data_path('./spacy_data')`

Problem was, when including a module in a subdirectory, that module was overriding to a "relative" path that wasn't relative to where the originating script was being run from. Just had to change it to set the right path to the spacy data and then everything was fine.
